frontend/i18next.scanner/en/translation.json

```diff
@@ -165,8 +165,6 @@
   "(protocole WebDAV)": "(WebDAV protocol)",
   "Administration": "Administration",
   "Agendas": "Agendas",
-  "See space": "See space",
```

Comment:
"Show space" would perhaps be better?

Reply:
I deleted this translations, I put "Actions" as tooltip (like in the action menu from All contents).